benchmarks/benchmark_serving.py

```diff
@@ -462,6 +462,7 @@ async def benchmark(
             logprobs=logprobs,
             best_of=best_of,
             multi_modal_content=mm_content,
+            ignore_eos=ignore_eos,
```

Comment:
Also add it to the other two `RequestFuncInput` in line 434 and 475

Reply:
I feel this is unnecessary since `start_profile`/`stop_profile` method will not actually trigger LLM inference.